engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -5761,11 +5767,18 @@ void demon_hunter_t::create_buffs()
                          set_bonuses.t30_havoc_2pc->ok() ? set_bonuses.t30_havoc_2pc_buff : spell_data_t::not_found() )
           ->set_default_value_from_effect_type( A_MOD_TOTAL_STAT_PERCENTAGE )
           ->set_pct_buff_type( STAT_PCT_BUFF_AGILITY );
-  
+
   buff.t30_havoc_4pc =
       make_buff<buff_t>( this, "seething_potential",
                          set_bonuses.t30_havoc_4pc->ok() ? set_bonuses.t30_havoc_4pc_buff : spell_data_t::not_found() )
           ->set_default_value_from_effect( 1 );
+
+  buff.t30_vengeance_2pc = make_buff<buff_t>(
+      this, "fires_of_fel", set_bonuses.t30_vengeance_2pc->ok() ? set_bonuses.t30_vengeance_2pc_buff : spell_data_t::not_found() )
+      ->set_default_value_from_effect( 1 )
+      ->add_invalidate( CACHE_PLAYER_DAMAGE_MULTIPLIER );
+  buff.t30_vengeance_4pc = make_buff<buff_t>(
```

Comment:
As a note, the set bonus passive "#3 (id=1074773) : Apply Aura (6) | Modify Spell School (220)" should be able to be done with `apply_affecting_aura( set_bonuses.t30_vengeance_4pc )` in the normal passive application spot to do the school modification.